prime-router/metadata/hl7_mapping/common/datatype/organization-xon.yml

```diff
@@ -0,0 +1,49 @@
+elements:
+  - name: organization-xon-name
+    value: [ '%resource.name' ]
+    hl7Spec: [ '%{hl7XONField}-1' ]
+
+  - name: organization-xon-name-type
+    resource: '%resource.extension(%`rsext-organization-name-type`).value'
+    condition: '%resource.exists()'
+    constants:
+      cweFieldPath: '%{hl7XONField}-2'
+    schema: ce-coded-with-exceptions
+
+  - name: id-number
+    value: [ '%resource.extension(%`rsext-xon3-id-number`).value' ]
+    hl7Spec: [ '%{hl7XONField}-3' ]
+
+  - name: identifier-check-digit
+    value: [ '%resource.identifier.extension(%`rsext-check-digit`).value' ]
+    hl7Spec: [ '%{hl7XONField}-4' ]
+
+  - name: identifier-check-digit-scheme
+    value: [ '%resource.identifier.extension(%`rsext-check-digit-scheme`).value' ]
+    hl7Spec: [ '%{hl7XONField}-5' ]
+
+  - name: xon-assigning-authority
+    resource: '%resource.identifier.extension(%`rsext-assigning-authority`)'
+    constants:
+      hl7HDField: '%{hl7XONField}-6'
+    schema: extensionAssigningAuthority-hd
```

Comment:
Should there be a condition here to check that the new resource exists before including `extensionAssigningAuthority-hd`?